src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public static Ball getRandomBall() {
```

Comment:
Don’t use static methods in your solution
Static methods are in general a bad practice. Let’s better create an instance of a class which method you want to call.
```suggestion
    public Ball getRandomBall() {
```